image.css

```diff
@@ -0,0 +1,109 @@
+:root{
+    --primary-color:turquoise;
+    --secondary-color:rgb(155, 173, 238);
+    --tertiary-color: skyblue;
+
+}
+
+body{
+    position: relative;
+    margin-left: 4px;
+    width: 100%;
+}
+
+.body{
+    width: 100%;
+}
+.container{
+    padding: 3px;
```

Comment:
below you are using re for padding and here px
try to keep consistent

Reply:
but vikram told us not rem in decimals